src/components/ActionsMenu/Item/ActionsMenuItem.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { TPAComponentsConsumer } from '../../TPAComponentsConfig';
+import styles from './ActionsMenuItem.st.css';
+import { Text } from '../../Text';
+
+function noop() {}
+
+export interface ActionsMenuItemProps {
+  prefixIcon?: React.ReactNode;
+  /** the item's content */
+  content: string;
+  /** the subtitle of the item */
+  subtitle?: string;
+  disabled?: boolean;
+  onClick(): void;
+}
+
+/** ActionsMenu */
+export class ActionsMenuItem extends React.Component<ActionsMenuItemProps> {
+  static displayName = 'ActionsMenu.Item';
+
+  render() {
+    const {
+      content,
+      prefixIcon,
+      subtitle,
+      disabled,
+      onClick,
+      ...rest
+    } = this.props;
+
+    return (
+      <TPAComponentsConsumer>
+        {({ mobile }) => (
+          <li
+            key={content}
+            {...styles('root', { mobile, disabled }, rest)}
+            role="menuitem"
+            tabIndex={-1}
+            aria-disabled={disabled}
+            onClick={disabled ? noop : onClick}
```

Comment:
No need to use `noop`.
It's better to pass `undefined` so that no function will be attached to the event.